### PR TITLE
Inspec managed zone

### PR DESCRIPTION
**Pull request by slevenick**
Add InSpec resource for DNS managed zone(s)

-----------------------------------------------------------------
# [all]
## [inspec]
Add DNS managed zone(s)

### CODE REVIEW

**Comment by rileykarson** [comment]
https://github.com/GoogleCloudPlatform/magic-modules/pull/1195 might cause some churn on your end - I'd rebase to include that before sending this for review (it will be merged as soon as `merge-prs` finishes).

File: products/compute/inspec.yaml [hunk 1]
```diff
@@ -13,9 +13,6 @@
 
 --- !ruby/object:Provider::Inspec::Config
 manifest: !ruby/object:Provider::Inspec::Manifest
-  version: '0.1.0'
-  source: 'FIXME'
```
Comment (rambleraptor) [inline products/compute/inspec.yaml]: Could we figure this out? Seems like neither of these 'FIXME' properties are necessary.

Comment (slevenick) [reply products/compute/inspec.yaml]: That's why they have been removed

**Comment by slevenick** [comment]
Managed zone already exists in InSpec repo, don't want to overwrite it yet

**Pull request closed without merging**